service/controller/resource/deployment/worker_deployment.go

```diff
@@ -392,15 +392,17 @@ func newWorkerDeployments(customResource v1alpha1.KVMConfig, release *releasev1a
 		}
 		addCoreComponentsAnnotations(deployment, release)
 		// in case of adding additional properties to the worker deployment
-		addConditionalPropsToWorkerDeployment(deployment, capabilities)
+		addConditionalPropsToWorkerDeployment(deployment, customResource, i)
 
 		deployments = append(deployments, deployment)
 	}
 
 	return deployments, nil
 }
 
-func addConditionalPropsToWorkerDeployment(deployment *v1.Deployment, caps v1alpha1.KVMConfigSpecKVMNode) {
+func addConditionalPropsToWorkerDeployment(deployment *v1.Deployment, customObject v1alpha1.KVMConfig, workerIndex int) {
```

Comment:
How about renaming this `addHostVolumes` or something since that's all it does?

Reply:
it adds also the env var. but yeah naming is not the best. whatever thing works for you, works for me.

Reply:
fixed.